P5/Source/Guidelines/en/WD-NonStandardCharacters.xml

```diff
@@ -336,24 +308,40 @@ currently assigned to certain properties. When making such
 modifications, great care should be taken not to override standard
 informative properties for characters which already exist in the Unicode
 Standard, as documented in <ref target="#CH-eg-02">Freytag (2006)</ref>.</p>
-<p>The <gi>charProp</gi> element allows an encoder to supply information
-about a character or glyph. Where the information concerned relates to
-a property which has already been identified in the Unicode Standard, encoders are
-urged to use the appropriate Unicode property name. </p>
-<p>The following elements are used to record character properties:
+<!-- TODO phase 6 insert comment about validation of values -->
+<p>The <gi>unicodeProp</gi>, <gi>unihanProp</gi>, and
+<gi>localProp</gi> elements allow a TEI encoder to record information
+about a character or glyph:
+<specList>
+  <specDesc key="unicodeProp" atts="name value"/>
+  <specDesc key="unihanProp" atts="name value"/>
+  <specDesc key="localProp" atts="name value"/>
+</specList>
+</p>
+<p>Where the information concerned relates to a property which has
+already been identified in the Unicode Standard, use of the
+appropriate Unicode property name with <gi>unicodeProp</gi> is
+strongly encouraged. The use of available Unihan property names with
+<gi>unihanProp</gi> is similarly encouraged. Validation rules for
+property names <!-- and values --> according to Unicode conventions
```

Comment:
Why is "and values" commented out?

Reply:
Because validation for values comes later, property names are already validated.

Reply:
Right. But I don’t wonder if the whole sentence shouldn’t be re-worded. Something like “The schemas provided with these Guidelines limit the values of the `@name` attribute of `<unicodeProp>` and `<unihanProp>` to the current list of property names.”

Reply:
Then later, when we have some sort of checking for `@value`, we can add a sentence about “Furthermore, the schemas provide some constraints as to the possible values to be expressed on `@value`, depending on which property is being represented.” or whatever.

Either way, this is word smithing detail that can wait to next week. The (CSS?) change to the Stylesheets for long lists of values should happen before Monday.

Reply:
OK this makes sense to me and I see there is a TODO at the top of the paragraph. I'm keen on merging this PR today or tomorrow, so let's table @sydb 's suggestions for now.